index.html

```diff
@@ -191,48 +280,116 @@ <h2>Modifications to existing procedures</h2>
     <section>
       <h2>Methods</h2>
       <dl>
-        <dt><dfn data-dfn-for=RTCRtpTransceiver>setOfferedRtpHeaderExtensions</dfn></dt>
+        <dt><dfn data-dfn-for=RTCRtpTransceiver>getHeaderExtensionsToNegotiate</dfn></dt>
+        <dd>
+          <p>Execute the following steps:</p>
+          <ol>
+            <li>
+              <p>Let <var>transceiver</var> be the {{RTCRtpTransceiver}} that
+              this method was invoked on.</p>
+            </li>
+            <li>
+              <p>Return
+              <var>transceiver</var>.{{RTCRtpTransceiver/[[HeaderExtensionsToNegotiate]]}}.</p>
+            </li>
+          </ol>
+        </dd>
+      </dl>
+      <dl>
+        <dt><dfn data-dfn-for=RTCRtpTransceiver>setHeaderExtensionsToNegotiate</dfn></dt>
         <dd>
-          Execute the following steps:
+          <p>Execute the following steps:</p>
           <ol>
-            <li>Let <var>extlist</var> be the list of extensions in the argument.</li>
             <li>
-              For each extension <var>ext</var> in <var>extlist</var>, do the following:
+              <p>Let <var>transceiver</var> be the {{RTCRtpTransceiver}} that
+              this method was invoked on.</p>
+            </li>
+            <li>
+              <p>Let <var>extensions</var> be the first argument of this
+              method.</p>
+            </li>
+            <li>
```

Comment:
I don't agree with the algorithm here. The following one works nicely and leaves a bit more freedom to developers:
1/ for each extension in HeaderExtensionsToNegotiate set direction to stopped
2/ for each extension in extension find matching extension in HeaderExtensionsToNegotiate and set its direction to extension direction

Reply:
What part of the algorithm do you not agree with? Not being able to re-order or remove things? Why would you ever do that in a "get-modify-set" pattern? What's the benefit of deleting an element over setting the elements direction to stopped?

Reply:
I don't want apps to have hardcoded list of header extensions set because it is not very web compat friendly and it ignores browser defaults and implicitly turns off extensions that might be added by default in the future